pages/supervision/index.tsx

```diff
@@ -192,13 +303,36 @@ function Supervision({}: SupervisionProps) {
           />
           Réaffecter les jeunes
         </Button>
+
+        {idsJeunesSelected.length > 0 && (
+          <div className='relative row-start-3 col-start-3'>
+            <p className='text-base-medium text-center'>
+              {idsJeunesSelected.length} jeunes sélectionnés
```

Comment:
```suggestion
              {idsJeunesSelected.length} jeunes sélectionné(s)
```
 conditionner, si plus de 1, ajouter un s à sélectionné